src/xcode/ENA/ENA/Source/Services/Risk/Provider/Helper/DeviceTimeCheck.swift

```diff
@@ -0,0 +1,89 @@
+//
+// Corona-Warn-App
+//
+// SAP SE and all other contributors
+// copyright owners license this file to you under the Apache
+// License, Version 2.0 (the "License"); you may not use this
+// file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+import Foundation
+
+protocol DeviceTimeCheckProtocol {
+	func checkAndPersistDeviceTimeFlags(serverTime: Date, deviceTime: Date)
+	func resetDeviceTimeFlagsToDefault()
+}
+
+final class DeviceTimeCheck: DeviceTimeCheckProtocol {
+
+	// MARK: - Init
+
+	init(store: AppConfigCaching) {
+		self.store = store
+	}
+
+	// MARK: - Protocol DeviceTimeCheckProtocol
+
+	// MARK: - Internal
+
+	func checkAndPersistDeviceTimeFlags(serverTime: Date, deviceTime: Date) {
```

Comment:
I find `check` as a verb here not appropriate. Just from the name of the function, I would expect it to either return a boolean, throw an exception if some constraints are not met or show a notification to a user. None of these things seem to happen. In the end, it's really just `persist`, isn't it?
Btw, is `persist` something that you use regularly? When writing to `store`? Otherwise, something more common such as `update` might suffice.

Reply:
Good point! Renamed to "updateDeviceTimeFlags"